packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -0,0 +1,415 @@
+
+            if (values.address_state && !validLetterSymbol(values.address_state) && states_list?.length < 1) {
+                errors.address_state = validation_letter_symbol_message;
+            }
+
+            if (values.address_postcode) {
+                if (!validLength(values.address_postcode, { min: 0, max: 20 })) {
+                    errors.address_postcode = localize(
+                        'Please enter a {{field_name}} under {{max_number}} characters.',
+                        {
+                            field_name: localize('Postal/ZIP code'),
+                            max_number: 20,
+                            interpolation: { escapeValue: false },
+                        }
+                    );
+                } else if (!validPostCode(values.address_postcode)) {
+                    errors.address_postcode = localize('Only letters, numbers, space, and hyphen are allowed.');
+                }
+            }
+
+            return errors;
+        };
+
+        const showForm = (should_show_form: boolean) => {
+            setFormState({ ...form_state, ...{ should_show_form } });
+        };
+
+        const isFieldsIssue = (errors: FormikErrors<TFormInitialValues>, values: TFormInitialValues) =>
+            errors.address_line_1 ||
+            !values.address_line_1 ||
+            errors.address_line_2 ||
+            errors.address_city ||
+            !values.address_city ||
+            errors.address_state ||
+            errors.address_postcode;
+
+        const onSubmitValues = async (
+            values: TFormInitialValues,
+            { setStatus, setSubmitting }: FormikHelpers<TFormInitialValues>
+        ) => {
+            setStatus({ msg: '' });
+            setFormState({ ...form_state, ...{ is_btn_loading: true } });
+            const settings_values = { ...values };
+            if (values.address_state && states_list.length) {
+                settings_values.address_state = getLocation(states_list, values.address_state, 'value') || '';
+            }
+
+            const data = await WS.setSettings(settings_values);
+
+            if (data.error) {
+                setStatus({ msg: data.error.message });
+                setFormState({ ...form_state, ...{ is_btn_loading: false } });
+                setSubmitting(false);
+                return;
+            }
+
+            const get_settings = WS.authorized.storage.getSettings();
+
+            if (get_settings.error) {
+                setAPIInitialLoadError(get_settings.error.message);
+                setSubmitting(false);
+                return;
+            }
+            const { address_line_1, address_line_2, address_city, address_state, address_postcode } = get_settings;
+
+            setFormValues({
+                address_line_1,
+                address_line_2,
+                address_city,
+                address_state,
+                address_postcode,
+            });
+            setIsLoading(false);
```

Comment:
Is there any reason that we set is_loading to false before uploading document?

Reply:
@amina-deriv, I'm not sure. it was here before.  it is unnecessary, will remove